SDC-seed/PostgreSQL/listingCSVMaker.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+
+const writeListings = fs.createWriteStream('listingTable.csv');
+writeListings.write('listingId,reviews,avatar\n', 'utf8');
+
+const randomNum = (min, max) => (Math.floor(Math.random() * (max - min + 1) + min));
+
+function writeTenMillion(writer, encoding, cb) {
+  let i = 10000000;
+  let id = 0;
+
+  function write () {
+    let ok = true;
+
+    do {
+      i -= 1;
+      id += 1;
+      const listingId = id.toString().padStart(7, '0');
```

Comment:
Does the first argument to padStart have to be 8 for 8 total digits in 10,000,000?